File: leetcode/completed/2455.py
from typing import List

class Solution:
    def averageValue(self, nums: List[int]) -> int:
        cnt = sums = 0

        for num in nums:
            if num % 6 == 0:
                cnt += 1
                sums += num
        
        return sums // cnt if cnt else 0

File: leetcode/completed/2455.cpp
#include <vector>

using namespace std;

class Solution {
public:
    int averageValue(vector<int>& nums) {
        int sum = 0, cnt = 0;

        for (auto num : nums) {
            if (num % 6 == 0) {
                cnt++;
                sum += num;
            }
        }

        return cnt == 0 ? 0 : sum / cnt;

    }
};